Readme.md

```diff
@@ -28,10 +28,25 @@ See [ilastik.org](https://ilastik.org) for more info.
 
 ## Installation
 
+### Binary installation
+
 Go to the [download page][download-page], get the latest _non-beta_ version for your operating system, and follow the [installation instructions][how-to-install].
 If you are new to ilastik, we suggest to start from the [pixel classification workflow][pixel-classification].
 If you don't have a dataset to work with, download one of the example projects to get started.
 
+### Conda installation (experimental)
+
+ilastik is also available as a conda package on our `ilastik-forge` conda channel.
+We recommend using [mamba][mamba] instead of conda, for faster installation:
+
+```bash
+mamba create -n ilastik --override-channels -c pytorch -c ilastik-forge -c conda-forge ilastik
+
+# activate ilastik environment and start ilastik
+conda activate ilastik
+ilastik-app
```

Comment:
Not sure whether it's better, but users also can `conda run` from anywhere.
```suggestion
conda run -n ilastik --no-capture-output --live-stream ilastik-app
```

Reply:
that's interesting, but more to type...